### DISCUSSION TITLE
CYP3A expression profile

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1475] [Questions & Problems]
How can I incorporate a CYP3A enzyme expression profile file? No subdivision of subgroups. I queried the PK-sim expression profile file with two results, as follows: 
![2ec1d637680fe24b8d1af0bd292d591](https://user-images.githubusercontent.com/86552551/224472555-41eed4ee-640e-4927-9c32-0bf3e367ab8f.png)
Obviously, the first one is grayed out and unavailable. The second "symbol" is 3A4.

**Comment by HenrikCordes** [comment]
Dear @SCZ1174,
I'm not sure what you are looking for when you mean CYP3A. 
In humans, the most likely candidate when it comes to pharmacokinetics context you are looking for CYP3A4.  https://www.ncbi.nlm.nih.gov/gene/?term=CYP3A 

Regarding the entries, I believe it should not matter which entry you use. 
Did you check that both have the same expression profiles ?

When you follow the annotations links (click on the right blue numbers) you end up on the respektive NCBI sites: 
https://www.ncbi.nlm.nih.gov/gene/1574
https://www.ncbi.nlm.nih.gov/gene/1576 

Here you find that both entries point to CYP3A4. 
The ID 1574 however is outdated and replaced with the 1576 entry. 

Hope this helps,
Henrik
